src/utils/__tests__/user_registry.test.ts

```diff
@@ -0,0 +1,229 @@
+import { FiscalCode, NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { CertificationEnumEnum } from "../../../generated/userregistry-api/CertificationEnum";
+import { UserRegistryAPIClient } from "../../clients/userregistry_client";
+import { blurUser, getUserId, postUser } from '../user_registry';
+import { isNone, isSome } from "fp-ts/lib/Option";
+import { isRight, right } from "fp-ts/lib/Either";
+import * as config from "../config";
+
+const aUserId = "1000" as NonEmptyString;
+const aMockFiscalCode = 'AAAAAA00A00A000A' as FiscalCode
+const aMockValidId = {
+  id: aUserId
+}
+const aMockUser = {
+  certification: CertificationEnumEnum.SPID,
+  email: 'test@email.com',
+  externalId: 'AAAAAA00A00A000A',
+  name: 'Nome',
+  surname: 'Cognome',
+}
+const aCreatedMockUser = {
+  certification: CertificationEnumEnum.SPID,
+  email: 'test@email.com',
+  externalId: 'AAAAAA00A00A000A',
+  id: aUserId,
+  name: 'Nome',
+  surname: 'Cognome'
+}
+const createUserMock = (status) => {
```

Comment:
Maybe we can mock the entire client though :)

Reply:
Fix with bc0b763